enhancements/security/runtime-class-scc.md

```diff
@@ -105,31 +106,33 @@ However, if admins happened to have added them, they will also have to allow the
 This section describes how points will be assigned for `RequiredRuntimeClasses`.
-In general, a smaller set of RequiredRuntimeClasses is more restricted. For instance, a list that is a strict subset of another is more restrictive, and thus should have
+In general, a smaller set of RequiredRuntimeClasses is more restricted. For instance, a list that is shorter than another is more restrictive, and thus should have
 a lower score. A set that specifies `"*"` will have the maximum score added for `RequiredRuntimeClasses`.
 
-Taking the [current examples](#some-examples), there is no simple algorithm for how each runtime class is weighted against the others.
-For instance, an order could be argued (from least restrictive to most):
-`openshift-low-latency`, `"", `openshift-builder`, `openshift-sandboxed-containers`.
-- `openshift-low-latency` gives access to configuring the host in particular ways, and is definitely less restrictive than the other.
-- `openshift-builder` is certainly more restrictive than the default, because when `openshift-builder` is paired with any added capabilities, and the ability to run as root,
-it is more secure than the default paired with capabilities + run as root, as it is more secure to give a process capabilities/UID 0 in a user namespace, rather than
-the user namespace of the host.
-- `openshift-sandboxed-containers` are certainly the most restrictive, as it runs the process in an kernel separated container, protecting the host from a class of kernel
-vulnerabilities the default runtime is vulnerable to.
-
-Thus, a strategy similar to how [RunAsUser][run-as-user] could be implemented, where every new possible entry in AllowedRuntimeClasses must be given a point value.
-The one drawback of this approach is we must ensure every new runtime class must be given a new point value, and each entry must be validated against a known list of
-runtime classes to the SCC controller.
-A way to mitigate this drawback could be to have a default high value for an unknown runtime class, as it should assume an unknown runtime class gives unbounded power to
-a container.
+Thus, the following rules will apply:
+`points(X) = max(len(X), 3*p)`
+`points(["*"]) = 4*p`
+Where X is the `RequiredRuntimeClasses` field, and p is the relative weight against the weight of the other SCC fields.
+
+In other words, fields with length < 4 will be ordered by length. After 4 (including the special `"*"`), the set will have equal value, as it is unlikely sets
+will grow much larger than 3, and ordering them after that is less useful (only one runtime class in the set will be used anyway).
+
+`RequiredRuntimeClasses` relative weight to the other fields partially depends on the runtime classes that Openshift opts to support.
+To start, we will consider the case where Openshift supports `""`, `openshift-builder` and `openshift-sandboxed-containers` (notes on supporting
+`openshift-low-latency` in the [open questions](#open-questions) section).
```

Comment:
I think this is wrong, compare https://github.com/openshift/enhancements/pull/562#discussion_r559537004.